doc/src/build/cli-client.md

```diff
@@ -209,17 +210,19 @@ The following commands are supported by the Sui client:
     help                  Print this message or the help of the given subcommand(s)
     history               Print history
     merge-coin            Merge two coin objects into one coin
-    new-address           Generate new address and keypair
-    object                Get obj info
+    new-address           Generate new address and keypair with keypair scheme flag {ed25519
+                              | secp256k1}
+    object                Get object info
     objects               Obtain all objects owned by the address
     publish               Publish Move modules
     split-coin            Split a coin object into multiple coins
-    switch                Switch active address and network (e.g., Devnet, local RPC server)
+    switch                Switch active address and network(e.g., devnet, local rpc server)
```

Comment:
this is Devnet and RPC Server in the UI, should it match?

Reply:
on `--- Sui Console 0.9.0-df05544f ---`

<img width="831" alt="image" src="https://user-images.githubusercontent.com/4743676/190473019-c99826d1-b947-4902-9866-5618a3d6c02c.png">

looks like this is from https://github.com/MystenLabs/sui/blob/2bf1304757e212f337de076a5cfea29dd0f61aba/crates/sui/src/client_commands.rs#L52. happy to make the change there, a cursory glance through `git blame` suggests that the readme's phrasing was never committed (publicly, at least)